spec/lib/api/2.0/profiles-spec.js

```diff
@@ -233,19 +233,14 @@ describe('Http.Api.Profiles', function () {
                 .expect(500);
         });
 
-        it("should send a 400 for a known node with no active graph", function() {
+        it("should send a 200 for a known node with no active graph", function() {
             profileApiService.createNodeAndRunDiscovery.resolves({});
             profileApiService.getNode.resolves({});
             workflowApiService.findActiveGraphForTarget.resolves(null);
 
-            var errorMsg = "Error: Unable to locate active workflow or there are no bootSettings";
-
             return helper.request().get('/api/2.0/profiles')
                 .query({ macs: '00:00:de:ad:be:ef' })
-                .expect(400)
-                .then(function(resp) {
-                    expect(resp.body.message).to.equal(errorMsg);
-                });
+                .expect(200);
         });
 
```

Comment:
updated. it seems the lines are mismatch with original comments position. the logics is in setLookup, and the unittests cases are there, anyway update 'macs' and 'ips' in unittests to avoid confusion